packages/sdk/src/swap/services/ApiService.ts

```diff
@@ -24,12 +25,12 @@ export const getQuote: BackendQuery<
     }),
   };
 
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  const queryParams = new URLSearchParams(params as Record<string, any>);
-
-  const url = new URL(`/quote?${queryParams.toString()}`, baseUrl).toString();
-
-  const { data } = await axios.get<QuoteQueryResponse>(url, { signal });
+  const { data } = await axios.get<QuoteQueryResponse>('/quote', {
```

Comment:
we can use `axios.create` to create an instance that automatically sets the headers and then share it everywhere

Reply:
unfortunately we cant pass the `baseUrl` via `axios.create` bc we pass it to the methods, so the instance would only contain the headers. no strong opinion about it

Reply:
you can pass a base url: https://axios-http.com/docs/config_defaults

Reply:
i c